src/bignum.nr

```diff
@@ -129,9 +129,12 @@ where
         Self { limbs: limbs.as_array() }
     }
 
-    fn from_array(limbs: [Field; N]) -> Self {
-        Self { limbs }
-    }
+    // We avoid passing an array, because that would require the array length to be expressed as a generic param of BigNumTrait, and we don't want BigNumTrait to have any generic params, so that when we use BigNum, we can cleanly say `where BigNum: BigNumTrait`, without any extra generics being used.
```

Comment:
Intentionally leaving this commented-out code here, so that others don't get confused like I did (for ages)

Reply:
I've left a comment on the trait as that's where people are going to be looking when they add a new `from_array` equivalent.